web/modules/custom/usagov_ssg_postprocessing/src/EventSubscriber/TomeEventSubscriber.php

```diff
@@ -2,46 +2,59 @@
 
 namespace Drupal\usagov_ssg_postprocessing\EventSubscriber;
 
-use Drupal\Core\Site\Settings;
 use Drupal\tome_static\Event\ModifyHtmlEvent;
 use Drupal\tome_static\Event\CollectPathsEvent;
 use Drupal\tome_static\Event\TomeStaticEvents;
+use Drupal\Core\Site\Settings;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\Core\Language\LanguageManagerInterface;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 
 /**
- * Converts links to "/es" to "/es/".
+ * This event subscriber modifies static site generation:
+ *  - During path collection, removes excluded directories -- allowing us to
+ *    specify the omission of entire directories like jsonapi, node, etc.
+ *  - During HTML modification, adds a trailing slash to links to the "/es" path.
  *
  * @internal
  */
 class TomeEventSubscriber implements EventSubscriberInterface {
 
   /**
-   * Reacts to a collect paths event. Replaces any '/es/' (without sub-path) with '/es'
-   * Purpose: prevent a condition where Tome creates a redirect page at /es/index.html,
-   * presumably due to Drupal and the redirect module making a redirect from /es/ to /es.
+   * The entity type manager.
    *
-   * We don't need to replace /es/ with /es, because Tome will find the /es page as a node
-   * to process regardless.
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * The language manager.
    *
-   * @param \Drupal\tome_static\Event\CollectPathsEvent $event
-   *   The collect paths event.
+   * @var \Drupal\Core\Language\LanguageManagerInterface
    */
-  public function excludeEsSlash(CollectPathsEvent $event) {
```

Comment:
This was intended to solve the "es redirect" problem but it did not. Deleting it in favor of using an excluded path.